tests/test_memoize.py

```diff
@@ -1,19 +1,21 @@
 """
 Tests for zipline.utils.memoize.
 """
+from collections import defaultdict
+import gc
 from unittest import TestCase
 
-from zipline.utils.memoize import remember_last
+from zipline.utils.memoize import weak_lru_cache, remember_last
 
 
 class TestRememberLast(TestCase):
 
-    def test_remember_last(self):
+    def test_weak_lru_cache(self):
 
         # Store the count in a list so we can mutate it from inside `func`.
         call_count = [0]
 
-        @remember_last
```

Comment:
changing this implies that we will also need to change all the usages of `remember_last`, or is it only ever used as on instance methods?